docs/self-managed/about-self-managed.md

```diff
@@ -29,6 +29,7 @@ The following components are available for Camunda 8 Self-Managed:
 - Connectors
 - Optimize
 - Identity (not available in Camunda 8 SaaS)
+- Console [<span class="badge badge--enterprise-only">Enterprise only</span>](../../reference/licenses/#web-modeler)
```

Comment:
@urbanisierung the link anchor should be `#console` or `#web-modeler` is intended?

Reply:
Good catch @aabouzaid! fixed it.